src/axom/inlet/Table.cpp

```diff
@@ -13,6 +13,9 @@ namespace axom
 {
 namespace inlet
 {
+const std::string Table::INTERNAL_ARRAY_GROUP = "_inlet_array";
```

Comment:
Maybe `INTERNAL_ARRAY_GROUP_NAME` or `INTERNAL_ARRAY_GROUP_ID`? maybe drop INTERNAL not sure its needed.

INDEX to INDICES though...